ci/aws/aws-cluster.lokocfg.envsubst

```diff
@@ -59,6 +59,36 @@ EOF
     ]
   }
 
+  worker_pool "worker-pool-2" {
+    count         = 1
+    ssh_pubkeys   = ["$PUB_KEY"]
+    disk_size     = 30
+    instance_type = "t2.small"
+    spot_price    = "0.01"
+    labels        = "testing.io=yes,roleofnode=testing"
+    taints        = "nodeType=storage:NoSchedule"
+    lb_http_port  = 8080
```

Comment:
Can you please add a comment, that this is a workaround for a bug (insert issue link here)?